packages/fluttium_runner/lib/src/fluttium_runner.dart

```diff
@@ -102,15 +102,15 @@ class FluttiumRunner {
             final text = sanitize(e.text);
             switch (e.action) {
               case FluttiumAction.expectVisible:
-                return "await tester.expectVisible(r'$text');";
+                return 'await tester.expectVisible(r"$text");';
```

Comment:
While that does indeed fix this issue, wouldn't it raise the same one if we now would use `"` in our text selectors?

Reply:
Yes!
But can we pass a double quote from YAML file?

Reply:
We should be able to yes: `- tapOn: Enter " text` or maybe `- tapOn: Enter \" text` either would be valid I think

Reply:
Oh yes.. right 👍

I will rethink about this so... 🤔

Reply:
Is it mandatory to transform the input text in raw with the `r`? 
This is added for the Regex part on the `_findByText` method?

Reply:
Yea, to allow for writing regexs directly in the yaml they string needs to be `r`.

Reply:
Outch, so my new try does not help, please forget it 😬

Reply:
Haha no worries! I'll try to think on a solution as well later today, there is for sure a clean way to get it to work as we want 😉

Reply:
New try! 😅

I now use the multi-line notation. This allow us to use the raw string `r` notation, and this will escape the single quote, just as the backslash, but will not be affected by the `r`.

Reply:
Hmm interesting, we should still sanitize string but then do `.replace("'''", "\'''");` to ensure that `'''` still works as well

Reply:
👌